azure-pipelines.yml

```
@@ -118,46 +115,48 @@ stages:
             parameters:
               environment: ${{ parameters.env }}
               location: ${{ parameters.location }}
+              serviceConnection: $(serviceConnection)
               stack: "07-network-rg"
               project: $(project)
-              tfversion: $(tfversion)
               tfInitSub: ${{ variables.tfInitSub }}
-              builtFrom: $(Build.Repository.Name)
-              product: ${{ variables.product }}
+              product: $(product)
+              action: ${{ parameters.action }}
+
 
       - job: DeployInfrastructure
         dependsOn: NetworkRg
         steps:
-          - template: pipeline-steps/deploy-network.yaml
+          - template: steps/set-build-repo-suffix-env-var.yaml@cnp-azuredevops-libraries           
+          - template: steps/terraform.yaml@cnp-azuredevops-libraries
             parameters:
-              environment: ${{ parameters.env }}
+              overrideAction: ${{ parameters.action }}
+              environment: ${{ parameters.env  }}
+              component: "01-network"
+              serviceConnection: $(serviceConnection)
```

see https://github.com/hmcts/aks-cft-deploy/blob/d9918238c5c33c00e2a8107a0fde760de6c9a534/azure-pipelines.yml#L174
```suggestion
              serviceConnection: OPS-APPROVAL-GATE-MGMT-ENVS-CNP
```